CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 **Language**:
 
+- _Breaking:_ Case syntax now uses brackets `[]` rather than braces `{}`.
+
+  To convert previous PRQL queries to this new syntax simply change
+  `case { ... }` to `case [ ... ]`.
+
```

Comment:
This should move to 0.10.0.

Reply:
I've moved to a new unreleased section, though will probably get conflicts that need fixing when #3477 is in.